docs/doxygen/Doxyfile

```diff
@@ -26,7 +26,7 @@ INPUT = ../../src
 
 MACRO_EXPANSION = YES
 
-OUTPUT_DIRECTORY = docBin
+OUTPUT_DIRECTORY = .
```

Comment:
The `OUTPUT_DIRECTORY` should be set to a directory in the build directory. Only the final documentation should be in the source directory.

Reply:
Addressed in [Place doxygen output in docs build directory](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/2472/commits/cb6563d0a45e194f1df1e8f0e2b10ead921963af)

Moves the doxygen output to the documentation build directory